### PR TITLE
CVE-2023-36884 - Microsoft Office and Windows HTML RCE

### PR DESCRIPTION
**Pull request by melamedbn**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
A new pack for handling CVE-2023-36884 - Microsoft Office and Windows HTML RCE.

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/CVE_2023_36884_-_Microsoft_Office_and_Windows_RCE/Playbooks/playbook-CVE-2023-36884_-_Microsoft_Office_and_Windows_HTML_RCE.yml [hunk 1]
```diff
@@ -0,0 +1,2214 @@
...
+                root: modules
+                filters:
+                - - operator: isEqualString
+                    left:
+                      value:
+                        simple: modules.brand
+                      iscontext: true
+                    right:
+                      value:
+                        simple: QRadar_v2
```
Comment (ssokolovich) [inline Packs/CVE_2023_36884_-_Microsoft_Office_and_Windows_RCE/Playbooks/playbook-CVE-2023-36884_-_Microsoft_Office_and_Windows_HTML_RCE.yml:575]: Why v2 and not v3?

File: Packs/CVE_2023_36884_-_Microsoft_Office_and_Windows_RCE/Playbooks/playbook-CVE-2023-36884_-_Microsoft_Office_and_Windows_HTML_RCE.yml [hunk 2]
```diff
@@ -0,0 +1,2214 @@
...
+        simple: XSOAR Remediation - Malicious URLs
+      DomainToBlock:
+        complex:
+          root: Domain
+          accessor: Name
+          transformers:
+          - operator: uniq
+      EmailToBlock:
+        complex:
+          root: DBotScore
```
Comment (ssokolovich) [inline Packs/CVE_2023_36884_-_Microsoft_Office_and_Windows_RCE/Playbooks/playbook-CVE-2023-36884_-_Microsoft_Office_and_Windows_HTML_RCE.yml:1202]: no need for this

File: Packs/CVE_2023_36884_-_Microsoft_Office_and_Windows_RCE/Playbooks/playbook-CVE-2023-36884_-_Microsoft_Office_and_Windows_HTML_RCE.yml [hunk 3]
```diff
@@ -0,0 +1,2214 @@
...
+        complex:
+          root: IP
+          accessor: Address
+          transformers:
+          - operator: uniq
+      InputEnrichment:
+        simple: "False"
+      MD5:
+        complex:
+          root: DBotScore
```
Comment (ssokolovich) [inline Packs/CVE_2023_36884_-_Microsoft_Office_and_Windows_RCE/Playbooks/playbook-CVE-2023-36884_-_Microsoft_Office_and_Windows_HTML_RCE.yml:1239]: no need

File: Packs/CVE_2023_36884_-_Microsoft_Office_and_Windows_RCE/Playbooks/playbook-CVE-2023-36884_-_Microsoft_Office_and_Windows_HTML_RCE.yml [hunk 4]
```diff
@@ -0,0 +1,2214 @@
...
+        simple: XSOAR - Block Indicators playbook - ${incident.id}
+      SHA256:
+        complex:
+          root: File
+          accessor: SHA256
+          transformers:
+          - operator: uniq
+      URL:
+        complex:
+          root: DBotScore
```
Comment (ssokolovich) [inline Packs/CVE_2023_36884_-_Microsoft_Office_and_Windows_RCE/Playbooks/playbook-CVE-2023-36884_-_Microsoft_Office_and_Windows_HTML_RCE.yml:1288]: Check if you need this

File: Packs/CVE_2023_36884_-_Microsoft_Office_and_Windows_RCE/Playbooks/playbook-CVE-2023-36884_-_Microsoft_Office_and_Windows_HTML_RCE.yml [hunk 5]
```diff
@@ -0,0 +1,2214 @@
...
+                value:
+                  simple: DBotScore.Score
+                iscontext: true
+              right:
+                value:
+                  simple: "3"
+          accessor: Indicator
+          transformers:
+          - operator: uniq
+      UserVerification:
```
Comment (ssokolovich) [inline Packs/CVE_2023_36884_-_Microsoft_Office_and_Windows_RCE/Playbooks/playbook-CVE-2023-36884_-_Microsoft_Office_and_Windows_HTML_RCE.yml:1310]: you mentioned that this should be automatic no?

File: Packs/CVE_2023_36884_-_Microsoft_Office_and_Windows_RCE/Playbooks/playbook-CVE-2023-36884_-_Microsoft_Office_and_Windows_HTML_RCE.yml [hunk 6]
```diff
@@ -0,0 +1,2214 @@
...
+        simple: "True"
+      Username:
+        complex:
+          root: DBotScore
```
Comment (ssokolovich) [inline Packs/CVE_2023_36884_-_Microsoft_Office_and_Windows_RCE/Playbooks/playbook-CVE-2023-36884_-_Microsoft_Office_and_Windows_HTML_RCE.yml:1314]: no need

File: Packs/CVE_2023_36884_-_Microsoft_Office_and_Windows_RCE/pack_metadata.json [hunk 7]
```diff
@@ -0,0 +1,41 @@
+{
+    "name": "CVE-2023-36884 - Microsoft Office and Windows HTML RCE",
+    "description": "This pack handles CVE-2023-36884 - Microsoft Office and Windows HTML RCE vulnerability",
+    "support": "xsoar",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "categories": [
+        "Case Management"
```
[the remark this answers is not in the document]
Comment (melamedbn) [reply Packs/CVE_2023_36884_-_Microsoft_Office_and_Windows_RCE/pack_metadata.json:10]: Yes.

File: Packs/CVE_2023_36884_-_Microsoft_Office_and_Windows_RCE/Playbooks/playbook-CVE-2023-36884_-_Microsoft_Office_and_Windows_HTML_RCE.yml (hunk 5 above)
Comment (melamedbn) [reply Packs/CVE_2023_36884_-_Microsoft_Office_and_Windows_RCE/Playbooks/playbook-CVE-2023-36884_-_Microsoft_Office_and_Windows_HTML_RCE.yml:1310]: Fixed

**Review by ssokolovich — APPROVED** [review]

**Pull request merged**